src/test/resources/outputs/spark/operators/between.sql

```diff
@@ -1,23 +1,23 @@
 --#[between-00]
 -- between(decimal, int32, int32)
-SELECT `T_DECIMALS`.`da` AS `da` FROM `default`.`T_DECIMALS` AS `T_DECIMALS` WHERE `T_DECIMALS`.`da` BETWEEN -1 AND 1;
```

Comment:
Each dialect had some failing `BETWEEN` tests following the upgrade to 0.14.6. These were unrelated to the `UNION` changes.